apps/cookbook/src/app/examples/card-example/examples/card-example-variant.scss

```
@@ -0,0 +1,29 @@
+@use '@kirbydesign/core/src/scss/utils';
+@use './card-example.shared.scss';
+
+:host {
+  container-type: inline-size;
+  display: flex;
```

Guess this should simply be:
```suggestion
  display: block;
```